clients/coredata/event.go

```diff
@@ -98,12 +93,7 @@ func (e *eventRestClient) requestEventSlice(urlSuffix string, ctx context.Contex
 
 // Helper method to request and decode an event
 func (e *eventRestClient) requestEvent(urlSuffix string, ctx context.Context) (models.Event, error) {
```

Comment:
Please create a new issue to refactor context.Context usage to make it idiomatic (i.e. `ctx context.Context` as first parameter in a method/function parameter list).  Note that this will break the module's API contracts.  Reference this comment in the new issue.